usaspending_api/awards/models_matviews.py

```diff
@@ -261,18 +245,10 @@ class Meta:
 
 
 class SummaryView(models.Model):
-
-    # Fields
     action_date = models.DateField(blank=True, null=True)
     fiscal_year = models.IntegerField()
     type = models.TextField(blank=True, null=True)
     pulled_from = models.TextField()
-    awarding_agency_id = models.TextField(blank=True, null=True)
-    awarding_agency_name = models.TextField(blank=True, null=True)
-    awarding_agency_abbr = models.TextField(blank=True, null=True)
-    funding_agency_id = models.TextField(blank=True, null=True)
-    funding_agency_name = models.TextField(blank=True, null=True)
-    funding_agency_abbr = models.TextField(blank=True, null=True)
```

Comment:
Looks good, is there a list or some documentation you're referring to for which fields should be removed?

Reply:
I don't have a nice list, the cols are comments in the matview SQL located in usaspending_api/database_scripts/matviews/

Reply:
That's good enough, just so we have something to refer to for this PR.